setup.py

```diff
@@ -5,16 +5,7 @@
 
 # python .\setup.py build_ext --inplace
 
-
-# Delayed import; https://stackoverflow.com/questions/37471313/setup-requires-with-cython
-try:
-    from Cython.Build import cythonize
-except ImportError:
-
-    def cythonize(*args, **kwargs):
-        from Cython.Build import cythonize
-
-        return cythonize(*args, **kwargs)
```

Comment:
This was not doing anything that fixes installation without cython installed.
The real fix was on pyproject.toml by adding cython as build dependency